File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/atts/schema/read/AttWithProc.java
package ru.citeck.ecos.records3.record.op.atts.schema.read;

import lombok.Data;
import ru.citeck.ecos.records3.record.op.atts.proc.AttProcessorDef;

import java.util.List;

@Data
public class AttWithProc {
    private final String attribute;
    private final List<AttProcessorDef> processors;
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/atts/service/schema/read/proc/AttWithProc.kt
package ru.citeck.ecos.records3.record.op.atts.service.schema.read.proc

import ru.citeck.ecos.records3.record.op.atts.service.proc.AttProcDef

data class AttWithProc(
    val attribute: String,
    val processors: List<AttProcDef>
)
